j-hannah/js-report
# Add load more

Addresses #2.

@sirkitree I've added the load more functionality here and done quite a bit of refactoring as well. I've tried to get rid of some of the "test" stuff I was working on as well as any cruft that was left over from the boilerplate.

I'm going to create a least one follow up issue to handle loading state so we can provide users soem indication that the request is being processed.

**Testing Instructions:**
1. Go to the home page and click on the "Load More" button at the bottom of the list of curated articles.
2. Confirm that at least 1 additional article loads. Once all articles have loaded, the button will disappear. You can confirm all articles have loaded by checking React DevTools. In the props for the HomePage component, you should see something like this:

<img width="220" alt="screenshot 2017-03-05 10 51 35" src="https://cloud.githubusercontent.com/assets/2105211/23588869/ecdaa422-0191-11e7-8e88-ddf8dcb7b20f.png">
